src/server/editorServices.ts

```diff
@@ -1304,10 +1304,8 @@ namespace ts.server {
         private getConfigFileNameForFile(info: ScriptInfo) {

```

Comment:
instead of duplicating code, can you please, make infoShouldBeOpen parameter must here?